src/App.js

```diff
@@ -1,25 +1,34 @@
 import Home from "./pages/home/Home";
-import {Route, Routes} from 'react-router-dom';
+import { Route, Routes } from "react-router-dom";
 import HomeWrapper from "./components/UI/HomeWrapper";
 import Like from "pages/liked/Like";
 import WatchLater from "pages/watchLater/WatchLater";
 import History from "pages/History/History";
 import Playlist from "pages/playlist/Playlist";
+import Signup from "components/Auth/Signup";
+import Login from "components/Auth/Login";
+
 
 function App() {
   return (
-  <>
-  <HomeWrapper>
-  <Routes>
-    <Route path="/" element={<Home/>}/>
-    <Route path="/liked" element={<Like/>}/>
-    <Route path="/watchlater" element={<WatchLater/>}/>
-    <Route path="/history" element={<History/>}/>
-    <Route path="playlist" element={<Playlist/>}/>
-   </Routes>
-   </HomeWrapper>
-  </>
-  )
+    <>
+      {
+        <HomeWrapper>
+          <Routes>
+            <Route path="/" element={<Home />} />
+
+            <Route path="/liked" element={<Like />} />
+            <Route path="/watchlater" element={<WatchLater />} />
+            <Route path="/history" element={<History />} />
+            <Route path="playlist" element={<Playlist />} />
+
+            <Route path="/signup" element={<Signup />} />
+            <Route path="/login" element={<Login />} />
+          </Routes>
```

Comment:
CODE Suggestion:
Instead of adding routes in the App.js file, you could move it to a separate file say AppRoutes.js and then add the component here. This way, in the future, if the number of routes increases your code will look cleaner, readable easy to understand.

Reply:
thanks!  for pointing it out !